atc/db/check.go

```diff
@@ -232,6 +240,8 @@ func (c *check) finish(status CheckStatus, checkError error) error {
 		return err
 	}
 
+	c.endTime = endTime
```

Comment:
is it relevant to test that now `Start()` and `finish()` mutate the `struct` adding their `startTime` and `endTime` respectively?

Reply:
Tests are updated. Thx!